apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -217,8 +217,7 @@ export const ProgressDot = Radium(React.createClass({
     const showUnplugged = isUnplugged && (courseOverviewPage || onCurrent);
     const outlineCurrent = courseOverviewPage && onCurrent;
     const smallDot = !courseOverviewPage && !onCurrent;
-    const showLevelName = courseOverviewPage &&
-      (level.kind === LevelKind.named_level || level.kind === LevelKind.peer_review);
+    const showLevelName = courseOverviewPage && !!level.name;
```

Comment:
peer_review levels will always have a name (https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/script.rb#L700), which is why we can replace the check for level.kind with !!level.name